scripts/download-artifact.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 OLDEST_PR_BRANCH_MERGE_COMMIT=$(
   git rev-list "$PREFIXED_PULL_REQUEST_BASE_BRANCH"..HEAD --merges --reverse |
-  grep -o -m 1 '\w\+'
+  grep -o -m 1 '\w\+' || echo ''
```

Comment:
Usually if I want a line to be allowed to fail, I'll use `|| true`. That or `|| :` seem to be the two popular idioms. I guess this works too though!

Reply:
In this case, I want the resulting variable to be an empty string, true.

Reply:
The result would be an empty string with either of these three options.